### ISSUE TITLE
Error when using --corpus opensubs

### ISSUE THREAD

**Issue by Sashweed** [Conchylicultor/DeepQA#40]
Conchylicultor, et al.

Brilliant work on this! The install (and run) instructions are the best I have come accross but Im getting an error when using the opensubs source (cornell works fine)

It unpacks the opensubs data files correctly (to 100%) then:
```
OpenSubtitles data files: 100%|
Unexpected error: <class 'xml.etree.ElementTree.ParseError'>
Traceback (most recent call last
  File "main.py", line 29, in <module>
    chatbot.main() files:  
  File "/home/xxxxxx/sashweed/deepqa/master/chatbot/chatbot.py", line 149, in 
    self.textData = TextData(self.args)
  File "/home/xxxxxx/sashweed/deepqa/master/chatbot/textdata.py", line 69, in __init__
    self.loadCorpus(self.samplesDir)
  File "/home/xxxxxx/sashweed/deepqa/master/chatbot/textdata.py", line 218, in loadCorpus
    opensubsData = OpensubsData(self.corpusDir
  File "/home/xxxxxx/sashweed/deepqa/master/chatbot/opensubsdata.py", line 36, in __init__
    self.conversations = self.loadConversations(dirName)
  File "/home/xxxxxx/sashweed/deepqa/master/chatbot/opensubsdata.py", line 50, in loadConversations
    doc = self.getXML(filepath)
  File "/home/xxxxxx/sashweed/deepqa/master/chatbot/opensubsdata.py", line 123, in getXML
    return ET.parse(tmp
  File "/usr/lib/python3.5/xml/etree/ElementTree.py", line 1184, in parse
    tree.parse(source, parser)|
  File "/usr/lib/python3.5/xml/etree/ElementTree.py", line 596, in parse
    self._root = parser._parse_whole(source)
xml.etree.ElementTree.ParseError: not well-formed (invalid token): line 1, column 13
xxxxxx@Eve:~/sashweed/deepqa/master$
```
from other seq2seq programs I have tested I'm guessing its a malformed char somewhere in the data set ????

NB: I have installed the nltk and downloaded the punkt as per the readme

appreciate any assistance

J

**Comment by eschnou** [comment]
Which corpus file are you using ? The english one ? Could you double check your folder structure ?

Mine is the following:

/DeepQA/data/opensubs/en/
├── 1937
│   └── 9690_218268_286967_shukujo_wa_nani_o_wasureta_ka.xml.gz
├── 1994
│   └── 16176_198024_262100_venus_in_furs.xml.gz
├── 1997
│   └── 5694_104669_143608_post_cotum_animal_triste.xml.gz
....

**Comment by Sashweed** [comment]
yep

jsalvin@Eve:~/sashweed/deepqa/master/data/opensubs/en/1937$ ls
9690_218268_286967_shukujo_wa_nani_o_wasureta_ka.xml.gz

**Comment by Conchylicultor** [comment]
I'm sorry, I can't reproduce the issue.

Try to display which file make the program crash. Otherwise, you can try to re-download the dataset, maybe the file is corrupted ?

Also why do you have a `master/` folder in `.../deepqa/master/data/...` ? Your path should looks like eschnou's one.

**Comment by Sashweed** [comment]
Yeah master/ was my fault. I'm new to this ... like VERY new ... I'm a security strategist and playing with this in my off time ... 5 weeks ago I had never built/logged onto a Ubuntu server let alone seen a .py file or used git. Stupidly when I did the git clone I added master on the end (thinking it would only pull the master branch ... turns out it doesn't, it just creates a dir)

She is working fine now!

Thanks again both of you, this has been a STEEP learning curve but I'm starting to get it with people like yourselves help.
